hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: minor
+
+This release adds `LiveServerTestCase` and `StaticLiveServerTestCase` for
```

Comment:
I've just checked [the preview docs build here](https://hypothesis--3106.org.readthedocs.build/en/3106/changes.html), and it looks like we need the full cross-reference syntax:

```suggestion
This release adds :class:`~hypothesis.extra.django.LiveServerTestCase`
and :class:`~hypothesis.extra.django.StaticLiveServerTestCase` for
```